server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -198,11 +198,13 @@ fn generate_response_requisition(
         // Default
         user_id: None,
         approval_status: None,
-        program_id: None,
         sent_datetime: None,
         finalised_datetime: None,
         colour: None,
         is_sync_update: false,
+        program_id: None, // TODO?
```

Comment:
@andreievg What should the behavior be for response requistions. I assume we copy these fields over?

Reply:
yes please, while you at it, can you do this one please: 

https://github.com/openmsupply/open-msupply/blob/8365309b9c05e9a384f25d3593fe9080410e6a10/server/service/src/processors/transfer/requisition/create_response_requisition.rs#L129-L137

I think i've noted that in this issue, basically we just want to check store preference and only change approval_status, so don't really need AuthorisationField